packages/graphql-playground-middleware-express/src/index.ts

```diff
@@ -15,12 +15,13 @@ export type ExpressPlaygroundMiddleware = (
 export type Register = (options: any) => ExpressPlaygroundMiddleware
 
 const express: Register = function voyagerExpress(options) {
-  const middlewareOptions: MiddlewareOptions = {
-    ...options,
-    version,
-  }
-
   return (req, res, next) => {
+    const middlewareOptions: MiddlewareOptions = {
+      ...options,
+      version,
+      isLocal: req.headers.host.includes('localhost'),
```

Comment:
set `isLocal` to `false` to have a glimpse of the flashy loading component in local mode